tests/test_graphql.py

```diff
@@ -234,6 +234,49 @@ def test_filter_product_by_attributes(client, product_in_stock):
     assert product_data['name'] == product_in_stock.name
 
 
+def test_sort_products(client, product_in_stock):
+    # set price of the first product
+    product_in_stock.price = Money('10.00', 'USD')
```

Comment:
The convention used in our tests (and by extension whole codebase) is explicit`currency=` for money's currency.

Reply:
Isn't this because `currency` was in fact the third parameter to `Price`?

Reply:
Yep, may be a remnant of those times that sticked around. May be good to do a follow-up PR that drops the superficial `currency=` from our codebase then?

Reply:
Yup! I think explicitly naming parameters is useful when either the order of arguments is not clear or the argument itself is a magic value (like `True` or `7`).